packages/carbon-web-components/src/components/tearsheet/tearsheet.scss

```
@@ -0,0 +1,238 @@
+    .#{$block-class}__header[has-navigation] {
+      padding-block-end: 0;
+    }
+
+    .#{$block-class}__container {
+      inline-size: 100%;
+
+      @include breakpoint(md) {
+        inline-size: calc(100% - (2 * #{$spacing-10}));
+      }
+    }
+
+    .#{$prefix}--modal-header__heading.#{$block-class}__heading {
+      @include type.type-style('heading-04');
+    }
+
+    .#{$block-class}__header[has-close-icon],
+    .#{$block-class}__header[has-slug] {
+      padding-inline-end: $spacing-11;
+    }
+
+    .#{$block-class}__header[has-close-icon][has-slug] {
+      padding-inline-end: calc(#{$spacing-11 + $spacing-09});
+    }
+
+    .#{$block-class}__header-navigation {
+      margin-inline-start: calc(-1 * #{$spacing-05});
+      max-block-size: $spacing-08; /* cds-tabs too tall */
+    }
+
+    .#{$block-class}__content {
+      // Revert background color overridden by Carbon's modal - https://github.com/carbon-design-system/carbon/blob/main/packages/styles/scss/components/modal/_modal.scss#L54
+      .#{$prefix}--pagination,
+      .#{$prefix}--pagination__control-buttons,
+      .#{$prefix}--text-input,
+      .#{$prefix}--text-area,
+      .#{$prefix}--search-input,
+      .#{$prefix}--select-input,
+      .#{$prefix}--dropdown,
+      .#{$prefix}--dropdown-list,
+      .#{$prefix}--number input[type='number'],
+      .#{$prefix}--date-picker__input {
+        background-color: $field;
+      }
+
+      .#{$prefix}--select--inline .#{$prefix}--select-input {
+        background-color: transparent;
+      }
+
+      // and restore the 'light' prop in case light fields are wanted
+      .#{$prefix}--text-input--light,
+    .#{$prefix}--text-area--light,
+    .#{$prefix}--search--light .#{$prefix}--search-input,
+    .#{$prefix}--select--light .#{$prefix}--select-input,
+    .#{$prefix}--dropdown--light,
+    .#{$prefix}--dropdown--light .#{$prefix}--dropdown-list,
+    /* stylelint-disable-next-line prettier/prettier */
+    .#{$prefix}--number--light input[type='number'],
+    .#{$prefix}--date-picker--light
+      .#{$prefix}--date-picker__input {
+        background-color: $field-02;
+      }
+    }
+
+    .#{$pkg-prefix}--action-set
+      .#{$pkg-prefix}--action-set__action-button.#{$pkg-prefix}--action-set__action-button--expressive {
+      block-size: $spacing-11;
+    }
+  }
+
+  .#{$block-class}__buttons {
+    @extend .#{$block-class}__button-container;
+    @extend .#{$block-class-action-set};
+
+    display: flex;
+    background: $background;
+    inline-size: 100%;
+
+    &[hidden] {
+      @extend .#{$prefix}--visually-hidden;
+
+      display: none;
+    }
+
+    /* stylelint-disable-next-line selector-type-no-unknown */
+    ::slotted(cds-button) {
+      @extend .#{$block-class-action-set}__action-button;
+
+      flex: 0 1 25%;
+      block-size: $spacing-11;
+      max-inline-size: to-rem(232px);
+    }
+
+    /* stylelint-disable-next-line selector-type-no-unknown */
+    ::slotted(cds-button[kind='ghost']) {
```

looks like a few places are missing the prefix
```suggestion
    ::slotted(`#{$prefix}-button[kind='ghost']`) {
```